src/components/Banner/BannerHeader/BannerHeader.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './BannerHeader.css';
+
+export const BannerHeader = () => {
+  return (
+    <div className="banner__header">
+      <h3 className="banner__header--yellow">၂၁ ကြိမ်မြောက်</h3>
+    </div>
+  )
+}
```

Comment:
- add prop title to BannerHeader component
Eg:
BannerHeader = ( { title } ) => {
   <h3>{title}</h3>
}